bundles/org.openhab.io.transport.modbus/src/main/java/org/openhab/io/transport/modbus/internal/SimpleStopWatch.java

```diff
@@ -38,8 +36,8 @@
     }
 
     @FunctionalInterface
-    public abstract interface RunnableWithModbusException {
-        public abstract void run() throws ModbusException;
+    public abstract interface RunnableWithException<T extends Throwable> {
```

Comment:
why Throwable instead of ModbusException?

Reply:
This made me look this a bit closer. The whole change with this interface with generic exception is unnecessary -- was related to a version of the code when I was working on this PR.

Removed the generics now, which means that there is actually no diff to `SimpleStopWatch.java` at all.